packages/clarity-decode/src/layout.ts

```diff
@@ -25,8 +25,9 @@ export function decode(tokens: Data.Token[]): LayoutEvent {
             for (let i = 2; i < tokens.length; i += 3) {
                 let region: Layout.RegionData = {
                     id: tokens[i] as number,
-                    state: tokens[i + 1] as number,
-                    name: tokens[i + 2] as string
+                    interactionState: tokens[i + 1] as number,
```

Comment:
Does this beak backwards compat. Could you put the new field at the end to keep it compatible?

Reply:
Because this is decoded as an array of tokens - so adding to end will not help us differentiate whether it was the id of the next region; I am keeping it in between and using the type to check how to decode this.